common/api/core-bentley.api.md

```diff
@@ -1475,6 +1475,13 @@ export enum RpcInterfaceStatus {
     Success = 0
 }
 
+// @public
+export interface SessionProps {
```

Comment:
why move this interface? Won't this (and other moves) be a breaking change?

Reply:
It was to avoid a circular dependency between common and rpc-common (which uses this...and still will after the larger refactor is finished.)

Reply:
Yes, the intrinsic “breaking change” of moving the rpc classes themselves is unavoidable here.

Reply:
but that's a breaking change. Why would common depend on rpc-common?

Reply:
It will have to in order to get the RpcInterface base type that its own RPC interfaces derive from.

Reply:
why do we need "rpc" packages at all? Why not make them part of common/frontend/backend packages that already exist. What value is there in separate packages?

Reply:
RPC doesn’t have anything to do with digital twins. It’s just a utility that we are using (and could ultimately replace wort something else. Separate packages make that clear and prevent more undesirable entanglement in the future.

Reply:
Are you targeting 4.0 with this PR? We can't introduce breaking changes in 3.x.

Reply:
Sounds like this might have to wait for 4, then.

I think we still want to get the backend node types out of common/rpc now though.

I’ll try to think up a bandaid for 3, then...